dist/index.html

```diff
@@ -505,10 +505,145 @@ <h2 class="causes__h1 text-c100 text-xl text-center font-bold">
         <img src="./assets/img/numbers/circle.png" alt="" />
       </div>
     </section>
+    <!-- events section start -->
+    <section class="flex upcoming-events relative">
+      <div class="events__bg-image absolute w-2/5">
+        <img
+          src="./assets/img/events/grey-blocks-upcoming-events.png"
+          alt="upcoming events Background Image"
+        />
+      </div>
```

Comment:
The image here is not content so you should add it as a background image with CSS, and as a side benefit you won't have to deal with that image showing on top of the heading. then you can push it down with negative bottom margin. I recommend using -mb-{value} from tailwind utility classes